examples/featherwing_keyboard_featherwing.py

```diff
@@ -20,13 +20,16 @@
 kbd_featherwing.neopixel[0] = 0x002244
 
 try:
-    f = open("/sd/tft_featherwing.txt", "w")
-    f.write("Blinka\nBlackberry Q10 Keyboard")
-    f.close()
+    with open(  # pylint: disable=unspecified-encoding
+        "/sd/tft_featherwing.txt", "w"
```

Comment:
I'm a bit confused here - it was my understanding that the `"w"` is the encoding. Same for the `"r"` below. I don't understand why Pylint is throwing this error here.

Reply:
Wait no, that's mode. You may be right about disabling org-wide. I'll think on it.